pkgs/tools/games/jpsxdec/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, jdk
+, jre ? jdk
+, ant
+, unoconv
+, makeWrapper
+, makeDesktopItem
+}:
+let
+  pname = "jpsxdec";
+  version = "1.05";
+
+  description = "Cross-platform PlayStation 1 audio and video converter";
+
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = pname;
+    icon = pname;
+    comment = description;
+    desktopName = "jPSXdec";
+    categories = "AudioVideo;Utility;";
+  };
+in
```

Comment:
Please move that into the rec.

Reply:
Which part specifically? There shouldn't be a rec, I forgot to remove it.